packages/bus-rdf-resolve-quad-pattern/lib/ActorRdfResolveQuadPattern.ts

```diff
@@ -14,6 +14,11 @@ export const KEY_CONTEXT_SOURCES = '@comunica/bus-rdf-resolve-quad-pattern:sourc
  * @value {IDataSource} A source.
  */
 export const KEY_CONTEXT_SOURCE = '@comunica/bus-rdf-resolve-quad-pattern:source';
+/**
+ * @type {string} Context entry for the authentication for a source.
+ * @value {string} "username:password"-pair.
+ */
+export const KEY_CONTEXT_AUTH = '@comunica/bus-rdf-resolve-quad-pattern:auth';
```

Comment:
```suggestion
export const KEY_CONTEXT_AUTH = '@comunica/bus-http:auth';
```

Reply:
Could you also add a context key shortcut here?: https://github.com/comunica/comunica/blob/master/packages/actor-init-sparql/components/Actor/Init/Sparql.jsonld#L108-L118